lib/loadCell/loadCell.h

```diff
@@ -1,40 +1,89 @@
 /**
  *
  * HX711 library for Arduino
- * https://github.com/bogde/HX711
- *
- * MIT License
- * (c) 2018 Bogdan Necula
  *
+ * Created by Bruce Deng, Feb 6, 2021.
 **/
+#ifndef __loadCell__
+#define __loadCell__
+
 #include <HX711.h>
 
 namespace LoadCell {
-		HX711 _lc1;
-		HX711 _lc2;
 
-		int calibrationValue1; // ======== SET THIS TO SOMETHING ===========
-		int calibrationValue2;
+		HX711 ** _lc_amp_boards;
+		int * _addrs;
+		int _numSensors;
+
+		int init(int numSensors, int * addrs, byte * dout_pins,
+									byte * sck_pins, float * calibration_vals) {
 
-		#define DOUT1 2
-		#define SCL1 3
+			_addrs = (int *) malloc(numSensors * sizeof(int));
+			_lc_amp_boards = (HX711 **) malloc(numSensors * sizeof(HX711));
 
-		#define DOUT2 4
-		#define SCL2 5
+			_numSensors = numSensors;
 
-    void init() {
-			_lc1.begin(DOUT1, SCL1);
-		  _lc1.set_scale(calibrationValue1); //This value is obtained by using the SparkFun_HX711_Calibration sketch
-		  _lc1.tare();	//Assuming there is no weight on the scale at start up, reset the scale to 0
+			for (int i = 0; i < numSensors; i++) {
+				_addrs[i] = addrs[i];
+				_lc_amp_boards[i] = new HX711();
 
-			_lc2.begin(DOUT2, SCL2);
-			_lc2.set_scale(calibrationValue2);
-			_lc2.tare();
+				if(!_lc_amp_boards[i]->wait_ready_retry()) {
+					Serial.println("timed out, loadCell: " + i);
+					return -1;
+				}
+
+				_lc_amp_boards[i]->begin(dout_pins[i], sck_pins[i]);
+				_lc_amp_boards[i]->set_scale(calibration_vals[i]);
+				_lc_amp_boards[i]->tare();
+
+			}
+			return 0;
 		}
 
-		void readLoad(float * data) {
-			data[0] = _lc1.get_units();
-			data[1] = _lc2.get_units();
-			data[2] = -1;
+		void readLoadCells(float *data) {
+			for (int i = 0; i < _numSensors; i++) {
+				data[i] = _lc_amp_boards[i]->get_units();
+			}
+			data[_numSensors] = -1;
 		}
+
+		int freeAllResources() {
+        free(_cryo_amp_boards);
+        free(_addrs);
+        return 0;
+    }
+
```

Comment:
`_lc_amp_boards` not `_cryo_amp_boards`

Also, you have some weird formatting / indenting going on. Can you fix that.